test/Datadog.Trace.ClrProfiler.IntegrationTests/AspNet/AspNetWebFormsTests.cs

```diff
@@ -23,7 +23,7 @@ public AspNetWebFormsTests(IisFixture iisFixture, ITestOutputHelper output)
             SetServiceVersion("1.0.0");
 
             _iisFixture = iisFixture;
-            _iisFixture.TryStartIis(this);
+            _iisFixture.TryStartIis(this, classicMode: false);
```

Comment:
We don't want to run this test in classic mode as well?

Reply:
This test uses OWIN, so it crashes in classic mode :(